fresh_mm_bot/src/commands/VoteMVP.ts

```
@@ -0,0 +1,38 @@
+            { $set: { 'players.$.mvpVoteId': mvpVoteId }, $inc: { version: 1 } }
+        );
+        if (result.modifiedCount === 0) {
+            console.log('Player mvp vote conflict, retrying');
+            setTimeout(() => {
+                setPlayerMvpVote({ playerId, matchNumber, client, mvpVoteId });
+            }, 1000);
+            return;
+        }
+
+        resolve(true);
+    });
```

Biome (2.4.10)

[error] Promise executor functions should not be `async`.

(lint/suspicious/noAsyncPromiseExecutor)